src/main/java/core/basesyntax/Structure.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Structure {
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+
+        Figure[] figures = new Figure[5];
+        figures[0] = figureSupplier.getDefaultFigure();
+        figures[1] = figureSupplier.getRandomFigure(2);
```

Comment:
The figure must be created randomly, without specifying a number.
Rewrite the logic of the getRandomFigure method, without a parameter.
And according to the condition of the problem, the figures must be created in a cycle, the first half is random values, the second is default.